Corona-Warn-App/src/test/java/de/rki/coronawarnapp/vaccination/core/VaccinationTestData.kt

```diff
@@ -1,263 +1,121 @@
 package de.rki.coronawarnapp.vaccination.core
 
+import de.rki.coronawarnapp.util.compression.inflate
 import de.rki.coronawarnapp.util.encoding.decodeBase45
-import de.rki.coronawarnapp.vaccination.core.common.RawCOSEObject
-import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateData
-import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateHeader
-import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateQRCode
-import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateV1
+import de.rki.coronawarnapp.vaccination.core.certificate.RawCOSEObject
+import de.rki.coronawarnapp.vaccination.core.certificate.VaccinationDGCV1
+import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateCOSEParser
 import de.rki.coronawarnapp.vaccination.core.repository.storage.ProofContainer
 import de.rki.coronawarnapp.vaccination.core.repository.storage.VaccinatedPersonData
 import de.rki.coronawarnapp.vaccination.core.repository.storage.VaccinationContainer
-import de.rki.coronawarnapp.vaccination.core.server.ProofCertificateV1
-import de.rki.coronawarnapp.vaccination.core.server.proof.ProofCertificateData
-import de.rki.coronawarnapp.vaccination.core.server.proof.ProofCertificateResponse
-import de.rki.coronawarnapp.vaccination.decoder.ZLIBDecompressor
+import de.rki.coronawarnapp.vaccination.core.server.proof.ProofCertificateCOSEParser
 import okio.ByteString.Companion.decodeBase64
 import okio.internal.commonAsUtf8ToByteArray
 import org.joda.time.Instant
-import org.joda.time.LocalDate
+import javax.inject.Inject
 
-object VaccinationTestData {
+class VaccinationTestData @Inject constructor(
+    private val vaccinationCertificateCOSEParser: VaccinationCertificateCOSEParser,
+    private val proofCertificateCOSEParser: ProofCertificateCOSEParser,
+) {
```

Comment:
The helper class for `TestData` is injected because we want the test data including the parsers to actually evaluate it. Makes it a bit easier to use in other tests.

For each container we need to set the parser:

```kotlin
  val personAVac1Container = VaccinationContainer(
        scannedAt = Instant.ofEpochMilli(1620062834471),
        vaccinationCertificateCOSE = personAVac1COSE,
    ).apply {
        parser = vaccinationCertificateCOSEParser
    }
```